config/ember-try.js

```diff
@@ -13,6 +13,9 @@ module.exports = function() {
         {
           name: 'ember-lts-2.18',
           npm: {
+            dependencies: {
+              'ember-test-selectors': '5.0.0',
+            },
```

Comment:
Is this needed if the default in `package.json` is `^5.0.0`, or does locking 2.18 to *exactly* `5.0.0` make a difference?

Reply:
It is needed exactly. For our 2.12 app at Addepar we use resolutions to pin it. I could use resolutions here as well, maybe.

Going to land this and move on though, I have a few more unblocking-other-people changes I think I can get in here quickly.

Reply:
nm, need to rebuild anyway so trying to drop it. Will try resolutions if it is needed.

Reply:
Well, seems like I was wrong. This must only be a limitation of 2.12! :tada: